pinot-core/src/main/java/org/apache/pinot/core/segment/name/NormalizedDateSegmentNameGenerator.java

```diff
@@ -44,11 +44,11 @@
   private final SimpleDateFormat _inputSDF;
 
   public NormalizedDateSegmentNameGenerator(String tableName, @Nullable String segmentNamePrefix,
-      @Nullable String excludeSequenceId, @Nullable String pushType, @Nullable String pushFrequency,
-      @Nullable String timeType, @Nullable String timeFormat) {
+      boolean excludeSequenceId, @Nullable String pushType, @Nullable String pushFrequency, @Nullable TimeUnit timeType,
```

Comment:
This constructor could be used by outside applications? If so, please use a new constructor as needed and translate

Reply:
I don't believe so, it should only be called by the segment generator to generate the segment name